src/wix/test/WixToolsetTest.Sdk/MsbuildFixture.cs

```diff
@@ -44,7 +44,6 @@ public void CanBuildSimpleBundle(BuildSystem buildSystem)
             }
         }
 
-        [Theory(Skip = "Test demonstrates failure")]
```

Comment:
I'm pretty sure the Theory` attribute is required for it to be recognized as a test, I would be surprised if this is actually running.

Reply:
Very possibly. I don't use `Theory` because the error reporting is difficult to diagnose. I'm retesting.